packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -531,7 +533,9 @@ const StartVisitForm: React.FC<StartVisitFormProps> = ({
               },
             );
 
-            const onVisitCreatedOrUpdatedRequests = [...visitFormCallbacks.values()].map((callbacks) =>
+
+            const onVisitCreatedOrUpdatedRequests = [...visitFormCallbacksRef.current.values()].map((callbacks) =>
+
```

Comment:
```suggestion
            const onVisitCreatedOrUpdatedRequests = [...visitFormCallbacksRef.current.values()].map((callbacks) =>
```

Please don't add lots of unnecessary whitespace.

Reply:
Please get rid of the extra whitespace elsewhere in the PR.